test/test_torch.py

```diff
@@ -4393,6 +4395,7 @@ def test_from_buffer(self):
         self.assertEqual(floats.size(), 1)
         self.assertEqual(floats[0], 2.25)
 
+    @unittest.skipIf(sys.platform == "win32", "TODO: need to fix this test case for Windows")
```

Comment:
Manually managing the lifecycle of `f` in this test case is not successful: when removing `f` with `os.remove(f.name)` it gives `PermissionError: [WinError 5] Access is denied`. The problem seems to be from `s1 = torch.FloatStorage.from_file(...)` which still has `f` open at the end of the test.

Reply:
Same with above. Better to use handle instead.

Reply:
torch.FloatStorage.from_file(...) seems to only take filename as first argument. We should probably skip this test for now.